modules/tinymce/src/plugins/autosave/demo/html/demo.html

```diff
@@ -10,6 +10,6 @@ <h2>Plugin: autosave Demo Page</h2>
       <textarea name="" id="" cols="30" rows="10" class="tinymce"></textarea>
     </div>
     <script src="../../../../../js/tinymce/tinymce.js"></script>
-    <script src="../../../../../scratch/demos/plugins/autosave/demo.js"></script>
+    <script src="../../../../../lib/plugins/autosave/demo/ts/demo/Demo.js"></script>
```

Comment:
This should have been the right path, as it's what we use everywhere else. Additionally the one in scratch is correct, as it has other changes made to it (eg is compiled by webpack so has hot reload enabled).

I suspect your `yarn start` may not have been running if you got an error with this. Either way, the lib path it's not the one that we should be using 🤷‍♀️

- [ ] Revert this please, as lib is the incorrect path.

Reply:
My apologies. I wasn't using `yarn start` to run the demo.